WalletWasabi.Fluent/Views/Wallets/Send/PrivacyWarningsView.axaml

```diff
@@ -133,7 +135,8 @@
           <Border Classes="warning">
             <DockPanel>
               <PathIcon />
-              <TextBlock Text="Transaction uses semi-private coins." />
+              <TextBlock Text="Transaction uses semi-private coins." 
+                         ToolTip.Tip="This transaction is not private enough because its privacy is as good as the used coins' lowest privacy. Coinjoin more to have (enough) private coins for this transaction." />
```

Comment:
```suggestion
                         ToolTip.Tip="This transaction is not private enough because it spends non-private coins. Coinjoin more to have (enough) private coins for this transaction." />
```

Reply:
This one is the semi-private coins case, so it could be: `because it spends not fully private coins`